Tiltfile

```diff
@@ -317,9 +317,14 @@ def flavors():
     for template in template_list:
         deploy_worker_templates(template, substitutions)
 
+    delete_all_workload_clusters = kubectl_cmd + " delete clusters --all --wait=false"
+
+    if "aks" in settings.get("kustomize_substitutions", {}).get("MGMT_CLUSTER_NAME", ""):
+        delete_all_workload_clusters += clear_aks_vnet_peerings()
+
     local_resource(
         name = "delete-all-workload-clusters",
-        cmd = kubectl_cmd + " delete clusters --all --wait=false",
+        cmd = ["sh", "-ec", delete_all_workload_clusters],
```

Comment:
Why do we need to wrap this in a shell command now? Just curious...easier to debug?

Reply:
Good point, we dont really need to wrap this in `sh`. 
`local_resource.cmd` ensures it runs with right arguments. Reference: https://docs.tilt.dev/api.html#api.local_resource

Updating..

Reply:
Updated via cfecb11767eecb4062bbd5d4fcb39afb67fd62d4